### PR TITLE
tamgamp.lv2: init at  unstable-2020-05-14

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/tamgamp.lv2/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  };
+
+  buildInputs = [
+    zita-resampler lv2
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/sadko4u/tamgamp.lv2;
```
Comment (evils) [inline pkgs/applications/audio/tamgamp.lv2/default.nix:21]: as per [RFC0045](https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md) URLs should be quoted

File: pkgs/applications/audio/tamgamp.lv2/default.nix [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+       - Mesa Dual Rectifier preamplifier (orange channel, less gain)
+       - Mesa Dual Rectifier preamplifier (red channel, more gain)
+       - Peavey 5150II crunch channel
+       - Peavey 5150II lead channel
+       - VOX AC-30 Brilliant channel
+       - VOX AC-30 normal channel
+     '';
+    maintainers = [ maintainers.magnetophon ];
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```
Comment (evils) [inline pkgs/applications/audio/tamgamp.lv2/default.nix:49]: the source comes with an LGPL-3.0 license, i'm not sure if that should be `lgpl3` or `lgpl3Plus`

Comment (magnetophon) [reply pkgs/applications/audio/tamgamp.lv2/default.nix]: The code says GPL3Plus, so I went with that for now, and [opened an issue](https://github.com/sadko4u/tamgamp.lv2/issues/11) to make sure.

Thanks for the review!

Comment (magnetophon) [reply pkgs/applications/audio/tamgamp.lv2/default.nix]: Properly fixed now.

**Pull request merged**